d3a_interface/schemas.py

```diff
@@ -1,4 +1,4 @@
-"""
+'''
```

Comment:
I think this came from a global "find and replace" in the file. For docstrings `"""` are generally used :D Nothing big anyway of course